ui/apps/platform/cypress/integration/risk/risk.test.js

```diff
@@ -161,10 +161,7 @@ describe('Risk page', () => {
             cy.get(RiskPageSelectors.search.searchLabels).should('not.exist');
         });
 
-        it('should have a single URL search param key/value pair in its search bar', function () {
-            if (hasFeatureFlag('ROX_POSTGRES_DATASTORE')) {
-                this.skip();
-            }
+        it('should have a single URL search param key/value pair in its search bar', () => {
```

Comment:
You can also remove skip from the next test:
`'should have multiple URL search param key/value pairs in its search bar'`

Reply:
Good point! Fixed